playbooks/bootstrap.yml

```diff
@@ -1,6 +1,40 @@
 ---
-- import_playbook: "./bootstrap_tower.yml"
-- import_playbook: "./bootstrap_integreatly.yml"
-- import_playbook: "./bootstrap_cluster_create.yml"
-- import_playbook: "./bootstrap_cluster_teardown.yml"
-- import_playbook: "./bootstrap_validation.yml"
\ No newline at end of file
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - include_vars:
+        ./roles/tower/defaults/main.yml
+
+    - name: "Confirmation prompt for bootstrapping without setting integreatly_version"
+      pause:
+        prompt: 'Please confirm you want to bootstrap tower with the master branch of configuration and credential repos (yes/no)'
+      register: master_bootstrap_prompt
+      when: not cicd and integreatly_version is not defined
```

Comment:
As we're now setting the `integreatly_version` in the defaults/main.yml we probably don't need the `and integreatly_version is not defined` section